Package.swift

```diff
@@ -3,7 +3,7 @@ import PackageDescription
 let package = Package(
     name: "VaporApp",
     dependencies: [
-        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 3)
+        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 4)
```

Comment:
Including the minor in the template allows users to choose when they upgrade to newer versions of Vapor. This is ideal since they might not want the newer or changed functionality.